--- gnutls-3.6.0/tests/Makefile.am~	2017-08-19 21:40:55.000000000 +0000
+++ gnutls-3.6.0/tests/Makefile.am	2017-08-22 18:20:01.701166167 +0000
@@ -150,7 +150,7 @@
 	 hostname-check-utf8 pkcs8-key-decode-encrypted priority-mix pkcs7 \
 	 send-data-before-handshake recv-data-before-handshake crt_inv_write \
 	 x509sign-verify-error rng-op-nonce rng-op-random rng-op-key x509-dn-decode-compat \
-	 ip-check mini-x509-ipaddr trust-store base64-raw random-art dhex509self \
+	 ip-check mini-x509-ipaddr base64-raw random-art dhex509self \
 	 dss-sig-val sign-pk-api tls-session-ext-override
 
 if HAVE_SECCOMP_TESTS
